xbmc/FileItem.h

```diff
@@ -555,6 +557,8 @@ class CFileItem : public CGUIListItem, public IArchivable, public ISerializable,
    */
   void FillMusicInfoTag(const std::shared_ptr<const PVR::CPVREpgInfoTag>& tag);
 
+  mutable std::optional<CURL> m_curlPath;
```

Comment:
May I suggest to rename the member to `m_urlPath` (omit the 'c') - reading the current name I was immediately thinking about a relationship to `libcurl` here...